lib/generators/sufia/work/templates/locale.en.yml.erb

```diff
@@ -1,4 +1,10 @@
 en:
+  sufia:
+    select_type:
+      <%= file_name %>:
+        name:               "<%= human_name.titleize %>"
+        description:        "<%= human_name %> works"
```

Comment:
@rotated8 what exactly should be documented?  That you'll find i18n templates generated into your `config/locales` and that you should customize them?